### PR TITLE
Update en_us.json

### PR DESCRIPTION
**Pull request by mikeharv**
Create new feedback strings for validating CSC Sprite Lab levels.
image (24)

This task will likely be repeated several times until the pilot launches: 
https://codedotorg.slack.com/archives/C03DBDN67B7/p1667490321349469
## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/i18n/spritelab/en_us.json [hunk 1]
```diff
@@ -222,5 +222,22 @@
...
   "whenToWhile": "Your sprite is moving in all directions, but not smoothly. Change `when` to `while` in each event block. \n <xml><block type=\"gamelab_keyPressed\"><title name=\"CONDITION\">while</title><title name=\"KEY\">???</title></block></xml>",
-  "workspaceAlertError": "Your program stopped because of a problem: {error}"
+  "workspaceAlertError": "Your program stopped because of a problem: {error}",
+  "cscLandmarkTouchAllSprites": "Use the arrow keys to move the tour guide to the landmarks.",
+  "createNewSprite": "Don't forget to make a new sprite.",
+  "noSpeech": "Your sprite didn't say anything yet. ",
+  "cscLandmarkStatueOfLiberty": "Make sure one of your sprites is the Statue of Liberty. ",
+  "cscLandmarkStatueOfLibertyUpperRight": "The Statue of Liberty is in the wrong place - it should be in New York in the upper-right corner of the map.",
+  "cscLandmarkTouchButNoSpeech": "It looks like your sprite reached a landmark, but now it needs to say something! Add a speech block to your event to make your sprite say something.",
+  "cscLandmarkAddAnotherSprite": "You're missing another landmark. Use a new sprite block with a different costume than the first. ",
```
Comment (breville) [inline apps/i18n/spritelab/en_us.json:232]: super nit: space at end.

File: apps/i18n/spritelab/en_us.json [hunk 2]
```diff
@@ -222,5 +222,22 @@
...
+  "cscLandmarkGoldenGateBridge": "Make sure one of your sprites the Golden Gate Bridge.",
```
Comment (breville) [inline apps/i18n/spritelab/en_us.json:233]: nit: missing "is"

File: apps/i18n/spritelab/en_us.json [hunk 3]
```diff
@@ -222,5 +222,22 @@
...
+  "cscLandmarkGoldenGateBridgeLeft": "The Golden Gate Bridge is in the wrong place - it should be in California on the left side of the map.",
+  "cscLandmarkChangeCostume": "Use a new sprite block with a different costume than the first. ",
+  "cscLandmarkBehavior": "Use a behavior to make your guide move with the arrow keys. ",
```
Comment (breville) [inline apps/i18n/spritelab/en_us.json:236]: super nit: spaces at ends.

**Review by breville — APPROVED** [review]

**Comment by mikeharv** [comment]
Thanks @breville . We're going to ahead and merge (after drone completes) in light of [this response](https://codedotorg.slack.com/archives/C03DBDN67B7/p1667494769452949?thread_ts=1667490321.349469&cid=C03DBDN67B7) from Dan:
> Thanks Brendan for catching the typos (and inspiring me to add a .trim() method to the spreadsheet magic to avoid some of these in the future)! In terms of next steps, here's what I'd prefer:
The main goal is to make sure these get deployed so we can verify levels work; small typos feel like something that should fall on the curriculum team to catch in our review process. So if they don't block this being merged (which they don't - Brendan still approved), then I vote we merge without the fixes.
And - I made those updates to the spreadsheet curriculum is maintaining, which means they'll get 'scooped' the next time we export as en_us.json. So Brendan's feedback will get captured when we post another version tomorrow.

**Pull request merged**